source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -347,6 +327,11 @@ void SXRMBAppController::setupExporterOptions()
 		AMAppControllerSupport::registerClass<SXRMB2DMapScanConfiguration, AMExporter2DAscii, AMExporterOptionGeneralAscii>(sxrmbExportOptions->id());
 }
 
+void SXRMBAppController::setupUserConfiguration()
```

Comment:
@dretrex the userConfiguration of SXRMB is initialized when the beamline connected signal is triggered. I believe that it is the same for the other beamlines as well. 

We should have consistent implementation across beamlines. I leave a prototype function here only because I think it will be better to work on this refactor for each beamline individually.